### PR TITLE
Related videos automation

### PR DESCRIPTION
**Pull request by ge0ffrey**
Automate landing pages tag adds relateded videos

Needed before @tarilabs copy pastes our structure for the drools website

### CODE REVIEW

File: content/blog/2020/10/09/FacilityLocationProblemQuickstart.adoc [hunk 1]
```diff
@@ -6,7 +6,7 @@ yurloc
 :jbake-tags: use case, facility location problem, quarkus
 :jbake-social_media_share_image: facilityLocationProblemSolved.png
 
-We have created a new https://github.com/kiegroup/optaplanner-quickstarts/tree/development/quarkus-facility-location[quickstart] that extends OptaPlanner use case portfolio with the https://en.wikipedia.org/wiki/Facility_location_problem[facility location problem] and also demonstrates building a cloud-native application using https://quarkus.io/[Quarkus], the new Java Native stack.
+We have created a new https://github.com/kiegroup/optaplanner-quickstarts/tree/stable/use-cases/facility-location[quickstart] that extends OptaPlanner use case portfolio with the https://en.wikipedia.org/wiki/Facility_location_problem[facility location problem] and also demonstrates building a cloud-native application using https://quarkus.io/[Quarkus], the new Java Native stack.
```
Comment (rsynek) [inline content/blog/2020/10/09/FacilityLocationProblemQuickstart.adoc:9]: This was already merged https://github.com/kiegroup/optaplanner-website/pull/371. Maybe try rebasing your PR.

File: content/blog/2021/03/25/BatchSolvingWithAmq.adoc [hunk 2]
```diff
@@ -10,7 +10,7 @@ If I want to solve many data sets of a planning problem every night, what archit
 In this article, we will take a look at how to use a transactional ActiveMQ queue in front of a set of stateless https://www.optaplanner.org[OptaPlanner] pods.
 Client applications can submit data sets to solve and listen to the resulting solutions without worrying about which OptaPlanner pod does the actual solving.
 
-The source code is available in the https://github.com/kiegroup/optaplanner-quickstarts/tree/development/activemq-quarkus-school-timetabling[activemq-quarkus-school-timetabling quickstart], along with the other https://github.com/kiegroup/optaplanner-quickstarts[optaplanner-quickstarts].
+The source code is available in the https://github.com/kiegroup/optaplanner-quickstarts/tree/stable/technology/java-activemq-quarkus[activemq-quarkus-school-timetabling quickstart], along with the other https://github.com/kiegroup/optaplanner-quickstarts[optaplanner-quickstarts].
```
Comment (rsynek) [inline content/blog/2021/03/25/BatchSolvingWithAmq.adoc:13]: ditto.

File: data/videos.yml [hunk 3]
```diff
@@ -95,7 +95,7 @@
   author: dmarrazzo
   date: 2020-09-02
   tags:
-    - task assigning
+    - task assignment
```
Comment (rsynek) [inline data/videos.yml:98]: Consistency: the module's name is "task assigning", see https://github.com/kiegroup/kogito-apps/tree/main/task-assigning"

Comment (ge0ffrey) [reply data/videos.yml:98]: Good point!
In the website, all other tags use "task assignment". So either the website changes or the quickstart module name changes.

Comment (ge0ffrey) [reply data/videos.yml:98]: I 'll do this in a separate PR.

Comment (ge0ffrey) [reply data/videos.yml:98]: Created this jira: https://issues.redhat.com/browse/PLANNER-2499

Note that "task assignment" is part of the landing page URL. Changing that is SEO sensitive (permanent 301 is a workaround)
The task assigning example should become a quickstarts in time, and then we can call it task assignment?

**Pull request merged**